### DISCUSSION TITLE
I entered Tailwind CSS classes in your React components and they are not taking effect

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12500] [Help] [answered]
Hi

I entered Tailwind CSS classes in your React components, and they are not taking effect. 

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/1925563e-7e5a-48db-8539-a84d780e3c98)

my output is:-

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/25ff3229-5b54-4b9c-baa2-a3ac4ae66fe4)

How to solve this issue.

Thanks
Minal P

**Comment by wongjn** [comment]
Consider checking you have followed [the installation guide](https://tailwindcss.com/docs/installation/framework-guides) most appropriate for your framework.

**Comment by Zehad-10** [comment, marked as answer]
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./src/**/*.{js,jsx,ts,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

Did you add this in your tailwind.config.js file?

**Reply by DEEPY-X7** [reply]
YES
